app/assets/js/rtMedia.backbone.js

```diff
@@ -497,7 +497,14 @@ jQuery( function( $ ) {
 			},
 			initialize: function( config ) {
 				this.uploader = new plupload.Uploader( config );
-				/*
+				
+				//current_url will fetch present URL
+				var current_url = document.URL, find = 'message';
+				if( -1 !== current_url.indexOf( find ) ){
```

Comment:
Don't create un-necessary variables. Here `find` var not used anywhere else.

Reply:
Please follow WordPress JS coding standards. Please follow it in your changes as of now.